### ISSUE TITLE
Lints not showing any warnings

### ISSUE THREAD

**Issue by amrgetment** [solid-software/solid_lints#163]
I tried the lints and it didn't work, what I am missing here?
I didn’t use include as I want one lint only

![image](https://github.com/solid-software/solid_lints/assets/149420647/62014009-3762-44f1-9a29-87fa140bd1ac)
![image](https://github.com/solid-software/solid_lints/assets/149420647/373c8c1d-b854-46f1-ac3b-36ca36873e80)
![image](https://github.com/solid-software/solid_lints/assets/149420647/0d45da68-ec87-49f4-a7a6-e063313df98c)

**Comment by amrgetment** [comment]
I tried this command and it didn’t show any issue too

`dart run custom_lint`

**Comment by illia-romanenko** [comment]
We'll probably need a complete code to reproduce the issue, could you please create some simple project with it?

**Comment by amrgetment** [comment]
Here is my repo, I made a class called dojo.dart with class name Foo and I used this lint prefer_match_file_name
https://github.com/amrgetment/flutter_web_error

**Comment by solid-vovabeloded** [comment]
@amrgetment, you should use the following notation to enable custom lint rules: 
```yaml 
custom_lint:
  rules:
  - prefer_match_file_name
```

instead of 

```yaml
custom_lint:
  - prefer_match_file_name
```

**Comment by amrgetment** [comment]
Thanks, works fine now
I wonder why do I have to use `dart run custom_lint` but not `dart analyze`
